docs/secure-connections/sauce-connect/installation.md

```diff
@@ -71,7 +71,7 @@ As an alternative to downloading the client, you can use the Sauce Connect Proxy
 
 ## Log File
 
-Once you've started using Sauce Connect Proxy, a log file will appear in your computer's directory. The log file name depends on whether [`--tunnel-name`](/dev/cli/sauce-connect-proxy/#--tunnel-name-or---tunnel-identifier) was used.
+Once you've started using Sauce Connect Proxy, a log file will appear in your computer's directory. The log file name depends on whether [`--tunnel-identifier`](/dev/cli/sauce-connect-proxy/#--tunnel-name-or---tunnel-identifier) was used.
```

Comment:
See [this comment](https://github.com/saucelabs/sauce-docs/pull/993/files#r784475982)